proposals/hasCrossSiteAncestor.md

```diff
@@ -104,17 +104,21 @@ string. A UUID of the document.
 
 | Required Combinations | Notes |
 |---|---|
-|frameId | frameId must be a non-zero value|
```

Comment:
A non-zero frameId is globally unique.

In a different extension API we agreed to support frameId without tabId, which enables matching frames that are not in tabs. The relevant discussion is at https://github.com/w3c/webextensions/issues/91

Reply:
Thank you for the additional context, I'm fine adding it back in.

Reply:
Resolving based on this [commit](https://github.com/w3c/webextensions/pull/712/commits/1c38656d9093979f71d44c1434341b367168fcb0)